doc/document/PortingToFreeRTOSAWS.md

```diff
@@ -0,0 +1,298 @@
+# FreeRTOS AWS porting guide
+
+## Introduction
+
+The FreeRTOS Cellular Library exposes the capability of cellular modems with AT command interface. This guide provides a method to integrate FreeRTOS Cellular Library with FreeRTOS AWS. We use windows simulator and BG96 cellular module as an example. FreeRTOS Cellular Library is sub-moduled in vendor’s porting folder. To communicate with cellular modem, an implementation of comm interface is required. There are some comm interface implementations can be referenced to reduced porting effort. An example implementation of secure sockets with FreeRTOS cellular APIs is also provided in this guide. FreeRTOS AWS demo application can make use of secure sockets cellular to connect to AWS IoT Core.

```

Comment:
can we use a quip doc to review this readme file?

I think we can try to make it more concise.